src/App.vue

```diff
@@ -153,6 +153,13 @@ export default {
     let config;
     try {
       config = await this.getConfig();
+      const path = (window.location.pathname != '/') ? window.location.pathname : null;
```

Comment:
Instead of using a path, maybe use a query string instead, which allows for subdirectory hosting.
```suggestion
      const path = (window.location.search.substring(1) != '') ? window.location.search.substring(1) : null;
```